.github/workflows/catkin_build_test.yml

```diff
@@ -1,41 +0,0 @@
-name: Catkin Build Test
-on:
-  push:
-    branches:
-    - 'master'
-  pull_request:
-    branches:
-    - '*'
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        config:
-          - {rosdistro: 'melodic', container: 'px4io/px4-dev-ros-melodic:2021-05-31'}
-          - {rosdistro: 'noetic', container: 'px4io/px4-dev-ros-noetic:2021-05-31'}
```

Comment:
One of reasons we want to use this container is to align the CI containers and check whether SITL is functional.